docs/earthfile/builtin-args.md

```diff
@@ -36,6 +36,8 @@ The following builtin args are available
 | `EARTHLY_GIT_ORIGIN_URL` | The git URL detected within the build context directory. If no git directory is detected, then the value is an empty string. Please note that this may be inconsistent, depending on whether an HTTPS or SSH URL was used. | `git@github.com:bar/buz.git` or `https://github.com/bar/buz.git` |
 | `EARTHLY_GIT_PROJECT_NAME` | The git project name from within the git URL detected within the build context directory. If no git directory is detected, then the value is an empty string. | `bar/buz` |
 | `EARTHLY_GIT_COMMIT_TIMESTAMP` | The timestamp, as unix seconds, of the git commit detected within the build context directory. If no git directory is detected, then the value is an empty string. | `1626881847` |
+| `EARTHLY_VERSION` | The version of Earthly currently running. | `v0.6.2` |
```

Comment:
It might be useful to add some sort of footnote such as:

```suggestion
| `EARTHLY_VERSION` * | The version of Earthly currently running. | `v0.6.2` |
```

Then down after this table we should say:

```
* requires `VERSION --earthly-version-arg 0.6`
```

to make it more clear to show that users must also include this version feature in order to get this ARG.